tests/kubeless-test-client/svcbind.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  namespace: kubeless-test
+  name: redis
+spec:
+  clusterServiceClassExternalName: redis
+  clusterServicePlanExternalName: micro
+  parameters:
+    redisPassword: VXU3ZVRkY2tVdlFVMkNHYlYwM2gK
+    resources:
+      requests:
+        memory: 96Mi
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  namespace: kubeless-test
+  name: redis-binding
+spec:
+  instanceRef:
+    name: redis
+---
+apiVersion: gateway.kyma.cx/v1alpha2
+kind: Api
+metadata:
+  namespace: kubeless-test
+  name: test-svcbind
+  labels:
+    function: test-svcbind
+spec:
+  authentication: []
+  hostname: test-svcbind.yfactory.sap.corp
```

Comment:
Probably **kyma.local** domain should be used instead of **yfactory.sap.corp**

Reply:
Fixed.